src/prymitive.ts

```diff
@@ -0,0 +1,120 @@
+class School {
+  directions:Direction[] = [];
+
+  addDirection(direction:Direction) {
+    this.directions.push(direction);
+  }
+}
+
+class Direction {
+  private _name: string;
+  levels:Level[]  = [];
+
+  constructor( name:string ) {
+    this._name = name;
+  }
+  get name():string {
+    return this._name;
+  }
+
+  addLevel(level:Level) {
+    this.levels.push(level);
+  }
+}
+
+class Level {
+  private _program:string
+  private _name: string;
+
+  groups:Group[] = [];
+
+  constructor( name:string,  program:string) {
+    this._name = name;
+    this._program = program;
+  }
+
+  get name():string {
+    return this._name;
+  }
+
+  get program():string {
+    return this._program;
+  }
+
+  addGroup(group:Group) {

```

Comment:
Please, add explicitly type annotation to the value that returns the `addGroup` function.